pkg/reconciler/common/prune.go

```diff
@@ -21,75 +21,199 @@ import (
 	"fmt"
 	"os"
 	"regexp"
+	"strconv"
 	"strings"
 
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
 	batchv1 "k8s.io/api/batch/v1"
-	v1beta1 "k8s.io/api/batch/v1beta1"
+	"k8s.io/api/batch/v1beta1"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"knative.dev/pkg/logging"
 )
 
 const (
-	tektonSA         = "tekton-pipelines-controller"
-	CronName         = "resource-pruner"
-	JobsTKNImageName = "IMAGE_JOB_PRUNER_TKN"
-	ownerAPIVer      = "operator.tekton.dev/v1alpha1"
-	ownerKind        = "TektonConfig"
+	tektonSA                    = "tekton-pipelines-controller"
+	CronName                    = "resource-pruner"
```

Comment:
let us make this `tektoncd-resorce-pruner`

Reply:
This could be done. I am only thinking of updates, is there any chance the cronjob name change will create any problem with the updates?

Reply:
Do we put annotation on the cronjob we create ? If yes, then we could/should probably not rely on the name, and thus changing it now wouldn't be a problem.
That said, where is this name use : inside `openshift-pipelines` (then it's fine to not have tektoncd prefix I would guess), or in any annotated namespace (then, yes, we should prefix it because we are not pruning all resources, just tektoncd ones).

Reply:
earlier label/annotation was not there. 
We will be adding a label this time. https://github.com/tektoncd/operator/pull/410#discussion_r712104343